code/modules/client/preference_setup/loadout/items/augments.dm

```diff
@@ -255,3 +255,10 @@
 	allowed_roles = list("Physician", "Surgeon", "Pharmacist", "First Responder", "Psychiatrist", "Medical Intern", "Medical Personnel", "Security Officer", "Warden", "Security Cadet", "Investigator", "Security Personnel", "Corporate Liaison", "Assistant", "Off-Duty Crew Member")
 	faction = "Private Military Contracting Group"
 	cost = 1
+
+/datum/gear/augment/translator
+	display_name = "universal translator"
+	description = "Augmentation that allows the user to understand other human languages, but not speak them."
+	path = /obj/item/organ/internal/augment/translator
+	cost = 1
```

Comment:
This is not wanted and goes against our current internal guidelines for languages / what we want in the future from them. Gotta remove this from the loadout, sorry.

Reply:
Oh, okay. I'll do it now